pre2p/handlers.go

```diff
@@ -0,0 +1,39 @@
+package pre2p
+
+import (
+	"io/ioutil"
+	"log"
+	"net"
+
+	pre2ptypes "pocket/pre2p/types"
+	"pocket/shared/types"
+
+	"google.golang.org/protobuf/proto"
+)
+
+func (m *p2pModule) handleNetworkMessage(conn net.Conn) {
+	defer conn.Close()
+
+	data, err := ioutil.ReadAll(conn)
+	if err != nil {
+		log.Println("Error reading from conn: ", err)
```

Comment:
Seeing some fmt.Print... and some log.Println

I know we don't yet have a standard, but seems odd to not standardize the usage in this PR